cadcutils/setup.cfg

```diff
@@ -36,7 +36,7 @@ edit_on_github = False
 github_project = opencadc/cadctools 
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
 install_requires = requests>=2.8 lxml>=3.7.0 six html2text
-version = 1.1.2
+version = 1.1.10
```

Comment:
Quite the version jump.  Did nothing in between matter?

Reply:
I branch off from an earlier commit and this was just the result of merging. So yeah, all the other versions have been released already, including 1.1.10.